curvegrid/multibaas-for-google-sheets
# [#19] Improve error handling

Part of #19 

- [x] Cell side error handling
- [x] Handing UI menu side error handling
- [x] Refactor codes

**Manual Tests**

- Create a new spreadsheet
- Go to `Add-ons > Get add-ons`
- Type `multibaas` in search box
- Install (wait for installing)
- Go to `Add-ons > MultiBaas for Google Sheets > Set Deployment ID` and set it
- Go to `Add-ons > MultiBaas for Google Sheets > Set API Key` and set it
- Check error messages on using UI menu and custom formula in the cell
